packages/paste-core/components/combobox/src/ComboboxItems.tsx

```diff
@@ -36,16 +38,17 @@ const ComboboxItems: React.FC<ComboboxItemsProps> = ({
         ))}
       </ComboboxListboxGroup>
     ) : (
-      <ComboboxListboxGroup>
+      <ComboboxListboxGroup element={element}>
         <li role="presentation" key="total-size" style={{height: rowVirtualizer!.totalSize}} />
         {rowVirtualizer!.virtualItems.map((virtualItem: VirtualItem) => {
           const itemIndex = virtualItem.index;
           const item = items[itemIndex];
           return (
             <ComboboxListboxOption
               // @ts-ignore
-              ref={(element) => virtualItem.measureRef(element)}
+              ref={(elementItem) => virtualItem.measureRef(elementItem)}
```

Comment:
I made this change so that I could use the element variable higher up in the ComboboxItems scope (for customization). Just wanted to make a note so that others can validate that this is an OK change.